app/javascript/controllers/normal_playback_controller.js

```diff
@@ -38,6 +43,7 @@ export default class extends PlaybackController {
       // if another track details is open, close it
       if (currentlyOpen) {
         currentlyOpen.closeDetails()
+        currentlyOpen = undefined
```

Comment:
Nice catch with this. We should prob go one step further and move the assignment into `closeDetails()` so that it conceptually parallels the assignment in `openDetails()`. Also, how do you feel about `false` vs. undefined?....

Reply:
@sudara I thought about that distinction and chose undefined because the assignment is an object if it exists, rather than a true. But I'm not too invested either way.